cmd/bosun/web/chart.go

```diff
@@ -165,6 +165,8 @@ func Graph(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interf
 	}, nil
 }
 
+//ExpGraph returns an svg graph.
+//the basename of requested svg file should be a base64 encoded expression
```

Comment:
Spaces after comment chars, i.e. `// Foo boo baz.`
Capitalization of "The" in the second sentence and fix missing period.